packages/react-router/src/router.ts

```diff
@@ -2558,15 +2558,10 @@ export class Router<
     )
 
     this.streamedKeys.add(key)
-    const children = `__TSR__.streamedValues['${key}'] = { value: ${this.serializer?.(this.options.transformer.stringify(value))}}`
+    const children = `__TSR__.streamedValues['${key}'] = { value: ${this.serializer?.(this.options.transformer.stringify(value))} }`
 
     this.injectHtml(
-      `<script class='tsr-once'>${children}${
-        process.env.NODE_ENV === 'development'
-          ? `; console.info(\`Injected From Server:
-        ${children}\`)`
-          : ''
-      }; __TSR__.cleanScripts()</script>`,
+      `<script class='tsr-once'>${children}; __TSR__.cleanScripts()</script>`,
```

Comment:
Do we need to remove the logging for this? Couldn't we just correctly escape the `console.log` statements instead?